lib/wizard/builder.rb

```diff
@@ -144,9 +144,10 @@ def build
           updater.update_setting(:base_font, updater.fields[:body_font])
           updater.update_setting(:heading_font, updater.fields[:heading_font])
 
-          if updater.fields[:homepage_style] == 'latest'
+          top_menu = SiteSetting.top_menu
```

Comment:
I am guilty of introducing this code here in the first place, but to fix, I think we need to find a way to prepend the chosen style in the dropdown to the `top_menu` setting. 

So, for example, if currently the `top_menu` setting is `categories|new|top|unread`, changing the Homepage style to `latest` should only prepend `latest`... right? Same for `categories`... because otherwise, this is resetting the other top menu items that have potentially been set by an admin.